mne/epochs.py

```diff
@@ -735,7 +735,7 @@ def _compute_mean_or_stderr(self, picks, mode='ave'):
         # pick channels
         if picks is None:
             picks = pick_types(evoked.info, meg=True, eeg=True, ref_meg=True,
-                               stim=False, eog=False, ecg=False,
+                               stim=False, eog=False, ecg=False, seeg=True,
                                emg=False, exclude=[])
```

Comment:
``'auto'`` is the default for ``pick_types`` and the docs say:
```
If True include CTF / 4D reference channels. If 'auto', the reference
        channels are only included if compensations are present.
```
So I'm asking if the ``_pick_data_channels`` should include them or use ``auto``.